### ISSUE TITLE
[ceph-volume] raw mode support

### ISSUE THREAD

**Issue by Anjingkegui** [ceph/ceph-container#1681]
**Feature Request**

Are there any similar features already existing:
no

What should the feature do:
ceph-volume raw was supported from 14.2.8. how about to have this in ceph-container.

What would be solved through this feature:
use ceph-volume raw in many container cases for osd bootstrap.

Does this have an impact on existing features:
no

**Environment**:
* Ceph version (14.2.9):

**Comment by dsavineau** [comment]
Do you have a specific scenario where ceph-volume raw would be a better solution than ceph-volume lvm ?

**Comment by travisn** [comment]
> ceph-volume raw was supported from 14.2.8. how about to have this in ceph-container.

@Anjingkegui `ceph-volume` is already included in the ceph container image and as you said it already has support for raw mode since 14.2.8. What else is required for your scenario?

**Comment by Anjingkegui** [comment]
> Do you have a specific scenario where ceph-volume raw would be a better solution than ceph-volume lvm ?

@dsavineau 
To set up osd in our internal Kubernetes, we met some problems of permission when using `ceph-volume lvm`, but we are fine with using `ceph-volume raw`.
Now we use this project to bundle the ceph image, but we can not use the entrypoint for osd, we use our script to set up osd by `ceph-volume raw`.

**Comment by Anjingkegui** [comment]
@travisn 
What I need is to have `ceph-voluem raw` to be part of the a unified entrypoint to set up osd, instead of using my script filled in this ceph image.

**Comment by willzhang** [comment]
yes

**Comment by PaulOh5** [comment]
I completely agree. Above all, the `ceph-disk` command used in the OSD scenarios is deprecated and causes errors in version after `reef`.
